### PR TITLE
Minor HTMl and code changes

### PR DESCRIPTION
**Pull request by TheMonDon**
Fix up some html
Fix a few more code with server.me to server.members.me
Fix channelData
Fix one more attachmentBuilder
Re-add tag to find member with depreciated note
Most of code changes is changing:
`		if (!server?.members) {
			this.callNextAction(cache);
			return;
	}`
to:
`	if (!server?.members) return this.callNextAction(cache);
`

### CODE REVIEW

**Review by almeidx — CHANGES_REQUESTED** [review]

**Comment by RobertBorghese** [comment]
@almeidx Thank you for those catches!! Definitely would've missed.😅 😅

@TheMonDon We really gotta try and keep some of these things separated, but good changes! Would it be okay to revert the `return this.callNextAction(cache);`? The way it was before was just more readable imo.

File: actions/edit_channel.js [hunk 1]
```diff
@@ -276,16 +276,17 @@ module.exports = {
...
 		}
+		channelData.reason = reason;
 
 		const channelStorage = parseInt(data.channel, 10);
 		const channelVarName = this.evalMessage(data.channelVarName, cache);
 		const channel = await this.getAnyChannel(channelStorage, channelVarName, cache);
 
 		if (Array.isArray(channel)) {
-			this.callListFunc(channel, "edit", [channelData, reason]).then(() => this.callNextAction(cache));
+			this.callListFunc(channel, "edit", [{ channelData }]).then(() => this.callNextAction(cache));
```
[the remark this answers is not in the document]
Comment (almeidx) [reply actions/edit_channel.js:286]: Not resolved

**Review by almeidx — APPROVED** [review]
I don't understand the random indentations on some of the options in the HTML but 🤷

**Comment by TheMonDon** [comment]
> I don't understand the random indentations on some of the options in the HTML but 🤷

It looks weird on github, but it is fixed in the actual code. Not sure why github displays it oddly.

**Comment by RobertBorghese** [comment]
I think it's cause it's mixing tabs and spaces. Your editor probably has tabs to be the same size as 4 spaces, but on github it's like 8 or something. No problemo tho, I was planning on going through and fixing all the spacing anyway. 👌

**Pull request merged**